libkbfs/mdserver_remote.go

```diff
@@ -387,6 +388,20 @@ func (md *MDServerRemote) ShouldRetryOnConnect(err error) bool {
 	return !inputCanceled
 }
 
+// CheckReachability implements the MDServer interface.
+func (md *MDServerRemote) CheckReachability(ctx context.Context) {
```

Comment:
Cool thanks, understood.  I guess my point is the opposite -- if the current connection is bad, but for some reason a _new_ connection will succeed easily, this won't actually fix KBFS's connectivity.  But I guess in that case we can just rely on the normal background pings.